src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -100,22 +100,27 @@ public function prepare_items() {
 		if ( 'scheduled' === $this->tab->get_slug() && ! empty( $_REQUEST['s'] ) ) {
 			// nonce check if search form submitted.
 			$nonce = isset( $_POST['s'] ) && isset( $_POST['aggregator']['nonce'] ) ? sanitize_text_field( $_POST['aggregator']['nonce'] ) :  '';
-			if ( wp_verify_nonce( $nonce,  'aggregator_' . $this->tab->get_slug() . '_request'  ) ) {
+			if ( isset( $_GET['s'] ) || wp_verify_nonce( $nonce,  'aggregator_' . $this->tab->get_slug() . '_request'  ) ) {
+				if ( false === filter_var( $_REQUEST['s'], FILTER_VALIDATE_URL ) ) {
+                    $search_term = sanitize_text_field( $_REQUEST['s'] );
+                }else{
+                    $search_term = sanitize_url( $_REQUEST['s'] );
+                }
```

Comment:
:no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: 83b57b41bc3aef052a325aee4af2607d</sub>